### ISSUE TITLE
Review A11y Project's hosted content

### ISSUE THREAD

**Issue by scottaohara** [a11yproject/a11yproject.com#669]
In an effort to ensure that The A11y Project's content is up-to-date, each hosted resource on the site should be reviewed to ensure code patterns are still relevant, links continue to work, and techniques are referencing the latest best practices.

If an article has been identified as needing an update, a new issue should be created and logged under the [Content Updates for 2019 Milestone](https://github.com/a11yproject/a11yproject.com/milestone/3) to track progress / request help from the community for updates.

If an article has recently been written, or received an update, then it can be checked off without further action.

If an article remains perfectly fine as-is, then it can be checked off without further action.

Assigning myself and @ericwbailey (sorry not sorry) as initial reviewers, but this issue is open to anyone who would like to participate.

The content to review is broken up by its parent topic grouping:

## How-tos
- [x] [Using skip navigation links](https://a11yproject.com/posts/skip-nav-links/)
- [x] [Create accessible data tables](https://a11yproject.com/posts/accessible-data-tables/)
- [x] [Using placeholder attributes](https://a11yproject.com/posts/placeholder-input-elements/)
- [x] [Hide content](https://a11yproject.com/posts/how-to-hide-content/)
- [x] [Using `title` attributes](https://a11yproject.com/posts/title-attributes/)
- [x] [Future proof your accessibility efforts.](https://a11yproject.com/posts/future-proof-your-accessibility/)
- [x] [Using caption services with HTML5 video](https://a11yproject.com/posts/using-caption-services-with-html5-video/)
- [x] [Using `role='application'`](https://a11yproject.com/posts/how-to-use-application-role/)

## Myths
- [x] [Screen readers don't use JavaScript](https://a11yproject.com/posts/myth-screen-readers-dont-use-javascript/)
- [x] [Accessibility is 'blind people'](https://a11yproject.com/posts/myth-accessibility-is-blind-people/)

## Quick tests
- [x] [Large touch targets](https://a11yproject.com/posts/large-touch-targets/)
- [x] [Check comprehension level](https://a11yproject.com/posts/check-comprehension-level/)
- [x] [Check contrast with your mobile device](https://a11yproject.com/posts/check-contrast-with-mobile-device/)
- [x] [Navigate using just your keyboard](https://a11yproject.com/posts/navigate-using-just-your-keyboard/)

## Quick tips
- [x] [Safari keyboard navigation](https://a11yproject.com/posts/safari-keyboard-navigation/)
- [x] [a11y and a brief numeronyms primer](https://a11yproject.com/posts/a11y-and-other-numeronyms/)
- [x] [Never remove CSS outlines](https://a11yproject.com/posts/never-remove-css-outlines/)
- [x] [Don't auto-play video, music and more](https://a11yproject.com/posts/never-use-auto-play/)
- [x] [Never use `maximum-scale=1.0`](https://a11yproject.com/posts/never-use-maximum-scale/)
- [x] [ARIA Landmark roles and HTML5 implicit mapping](https://a11yproject.com/posts/aria-landmark-roles/)
- [x] [Using alt text properly](https://a11yproject.com/posts/alt-text/)

## Assistive Technology
- [x] [Getting Started with NVDA](https://a11yproject.com/posts/getting-started-with-nvda/)
- [x] [Getting started with OS X VoiceOver](https://a11yproject.com/posts/getting-started-with-voiceover/)

## Background
- [x] [ARIA States](https://a11yproject.com/posts/ARIA-states/)
- [x] [ARIA Properties](https://a11yproject.com/posts/ARIA-properties/)
- [x] [Understanding colorblindness](https://a11yproject.com/posts/understanding-colourblindness/)
- [x] [What is Color Contrast?](https://a11yproject.com/posts/what-is-color-contrast/)
- [x] [Getting started with ARIA](https://a11yproject.com/posts/getting-started-aria/)
- [x] [A primer to Vestibular Disorders](https://a11yproject.com/posts/understanding-vestibular-disorders/)
- [x] [A primer to visual impairment](https://a11yproject.com/posts/understanding-visual-impairment/)

**Comment by scottaohara** [comment]
Issue #660 was created to update Getting started with ARIA

Using Placeholder attributes was reviewed and updated in July.

ARIA Landmark roles and HTML5 implicit mapping was reviewed and updated in July.

**Comment by ericwbailey** [comment]
Big-picture question: do we want to host technical content that is liable to become quickly outdated? I think some technical content is evergreen in nature, but certain topics could become out of date—and therefore bad advice—potentially in a few month's time. In particular, I'm thinking of things like ARIA support tables, where we'd have to ensure they were always inline with the state of the spec.

**Comment by scottaohara** [comment]
I was thinking on similar lines yesterday.

But I think if we keep articles focused to provide high level understanding, with some technical bits, we can then link off to resources to provide deeper-level technical instructions. That way we can walk the line of providing useful content for a wider audience (high level knowledge with some design, ux, and dev bits to allow for everyone to get some useful information), and then just make sure we date the links to resources to help make it clear how relevant the links may be.

**Comment by ericwbailey** [comment]
I like that. I think we should also specifically name it in [`CONTRIBUTING.md`](https://github.com/a11yproject/a11yproject.com/issues/668) and/or the [proposed article issue template](https://github.com/a11yproject/a11yproject.com/issues/668). It'll help prevent people from taking the time and effort of writing these kinds of things.

**Comment by ericwbailey** [comment]
@emilane, pinging you, since you did some really great writing work recently. I know it's the holiday season, but if you have some time, we'd really appreciate your talents here!

**Comment by emilane** [comment]
Thanks for looping me in! Looks like a great way to spend a slow Monday :)

**Comment by alexroseb** [comment]
Just reviewed the skipnav article, so I think all have been looked at and have issues created if they still need to be updated. I'm also going through and doing some of those updates!

**Comment by scottaohara** [comment]
thanks @alexroseb 

going to close this since that's the last of the content that needed an audit.  thanks!
